pages/common/git-ignore.md

```diff
@@ -1,12 +1,20 @@
 # git ignore
 
-> Generate .gitignore files from predefined templates.
-> More information: <https://docs.gitignore.io/install/command-line>.
+> Show/update `.gitignore` files.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-ignore>.
```

Comment:
This doesn't look like a core git command. I recommend a 2nd description line that includes this information - e.g. `Part of SOMETHING_HERE`:
```suggestion
> Show/update `.gitignore` files.
> Part of SOMMETHING_HERE. See also `git ignore-io`.
> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-ignore>.
```

Reply:
`git-extras`.  I'll update your suggestion.